cppcache/integration/test/RegisterKeysTest.cpp

```diff
@@ -112,6 +114,51 @@ TEST(RegisterKeysTest, RegisterAllWithCachingRegion) {
   }
 }
 
+TEST(RegisterKeysTest, RegisterAllWithConsistencyDisabled) {
+  Cluster cluster{LocatorCount{1}, ServerCount{1}};
+
+  cluster.start();
+
+  cluster.getGfsh()
+      .create()
+      .region()
+      .withName("region")
+      .withType("PARTITION")
+      .execute();
+
+  auto producer_cache = createTestCache();
+  auto listener_cache = createTestCache();
+
+  auto producer_region = [&cluster](Cache& cache) {
+    auto poolFactory = cache.getPoolManager().createFactory();
+    cluster.applyLocators(poolFactory);
+    poolFactory.create("default");
+    return setupProxyRegion(cache);
+  }(producer_cache);
+
+  auto listener_region = [&cluster](Cache& cache) {
+    auto poolFactory =
+        cache.getPoolManager().createFactory().setSubscriptionEnabled(true);
+    cluster.applyLocators(poolFactory);
+    poolFactory.create("default");
+    auto region = setupCachingProxyRegion(cache, false);
+    region->registerAllKeys();
+    return region;
+  }(listener_cache);
+
+  ASSERT_FALSE(listener_region->containsValueForKey("one"));
```

Comment:
No need to establish a new, empty region, that cannot possibly have any preloaded data in it is indeed empty.

Reply:
Yes, indeed, this is overtesting